eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -188,5 +188,7 @@ if ($GetGithubIoDocIndexFn -and (Test-Path "function:$GetGithubIoDocIndexFn"))
 }
 else
 {
-    LogWarning "The function 'GetGithubIoDocIndexFn' was not found."
+    LogWarning "The function for 'GetGithubIoDocIndexFn' was not found.`
+    Make sure it is present in eng/scripts/Language-Settings.ps1 and referenced in eng/common/scripts/common.ps1.`
+    See https://github.com/Azure/azure-sdk-tools/blob/master/doc/common/common_engsys.md#code-structure"
```

Comment:
Do you plan to include these doc updates in this PR?

Reply:
Am not sure what you mean by doc updates?

Reply:
Sorry for some reason on my computer the anchor link for code-structure isn't working but I do see there is already a section on that in the common_engsys.md file so those docs are already there.